ci-operator/config/open-cluster-management/submariner-addon/open-cluster-management-submariner-addon-main.yaml

```diff
@@ -6,9 +6,9 @@ base_images:
 binary_build_commands: make build --warn-undefined-variables
 build_root:
   image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.15
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.15-linux-amd64
```

Comment:
i think we should use this open-cluster-management/builder:go1.15-linux

Reply:
done